recipes/application.rb

```diff
@@ -1,6 +1,8 @@
 
 # include defaults
 include_recipe 'cyclesafe_chef'
+include_recipe 'runit'
+include_recipe 'python'
```

Comment:
Both of these includes are specific to the application side only. They do not need to be default across all cyclesafe nodes.